infrastructure/terraform/dns/README.md

```diff
@@ -0,0 +1,31 @@
+# DNS terraform config
+
+Managing the infrastructure for Wellcome Collection's DNS.
+
+Most services should handle their own DNS by importing what they need
+from here.
+
+## Purpose
+
+- To manage domains pointing to third-party services held outside of our
+  infrastructure.
+- Handling redirects.
+- To expose state data from the Routemaster account so other Terraform
+  provisionings can import that state. i.e. importing the Zone ID as
+  from the output `weco_hosted_zone_id`.
```

Comment:
Is it worth including some copy/paste Terraform you can use to get that? i.e. include the explicit `terraform and locals` blocks.

Reply:
I might extract this out into it's own module to make it easier, trying to make the examples made me realise this is a bit too fiddly. Going to merge as it's actually applied though.